src/style.css

```
@@ -0,0 +1,3 @@
+@import url(./styles/reset.css);
```

Add quotes in all places
```suggestion
@import url('./styles/reset.css');
```